cpu/nrf52/radio/nrf802154/nrf802154.c

```diff
@@ -120,6 +143,63 @@ static void _enable_tx(void)
     DEBUG("[nrf802154] Device state: TXIDLE\n");
 }
 
+/**
+ * @brief   Convert from dBm to the internal representation, when the
+ *          radio operates as a IEEE802.15.4 transceiver.
+ */
+static inline uint8_t _dbm_to_ieee802154_hwval(int8_t dbm)
+{
+    return ((dbm - NRF802154_ED_RSSIOFFS) / NRF802154_ED_RSSISCALE);
+}
+
+/**
+ * @brief   Convert from the internal representation to dBm, when the
+ *          radio operates as a IEEE802.15.4 transceiver.
+ */
+static inline int8_t _hwval_to_ieee802154_dbm(uint8_t hwval)
+{
+    return NRF802154_ED_RSSIOFFS + NRF802154_ED_RSSISCALE * hwval;
```

Comment:
Conversions seem reasonable, but I couldn't find references in the above mentioned document either.

Reply:
This equation is in the Figure 25 of https://github.com/RIOT-OS/RIOT/pull/13145#discussion_r367552584